docs/game_sdk/Store.md

```diff
@@ -0,0 +1,449 @@
+# Store
+
+> danger
+> The Discord Store is still in a beta period. All documentation and functionality can and will change.
+
+If your game has DLC or offers in-app purchases, this manager is for you! The Store Manager allows you to fetch a users' entitlements, as well as being notified when a user is granted an entitlement from a purchase flow for your game.
+
+## Application Test Mode
+
+With this new Store Manager comes a new fun toggle in the Discord app itself: Application Test Mode! You can find this toggle in your Discord user settings -> Appearance -> allllll the way at the bottom. If it's not there, make sure you have "Developer Mode" toggled on as well.
```

Comment:
This description seems good. Features will slowly trickle in for `Test Mode` as it was created immediately for IAP but there's other test flows that need support. Do you want me to notify you as those land? (For instance, about to open PR to allow devs to navigate to their unpublished store page in-app by clicking on the notice)...

Reply:
Yes, the more features the merrier!